docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -213,6 +213,14 @@ TestCafe will run fixtures whose names match the specified pattern.
 testcafe ie my-tests --fixture-grep *Page*
 ```
 
+### --element-timeout \<ms\>
+
+Specifies the amount of time, in milliseconds, allowed for a page element to become available before the test fails.
```

Comment:
So, are you saying that *available for use* or *available for performing actions on* is better?
Or do we need to rename the whole thing from *available* to a more strict descrption like *added to DOM*, *loaded* or whatever is technically correct in this situation IDK